pkg/cloud/openshift/cli.go

```diff
@@ -22,7 +22,7 @@ func InstallOc() error {
 		return err
 	}
 	binary := "oc"
-	fileName, flag, err := packages.ShouldInstallBinary(binary)
+	flag, err := packages.ShouldInstallBinary(binary)
```

Comment:
Same as above. Will this work with Windows? The downloaded tarball, does it contain an `oc.exe` or just plain `oc`?